src/CapAutomator.sol

```diff
@@ -2,7 +2,7 @@
 pragma solidity ^0.8.13;
 
 import { Ownable } from "openzeppelin-contracts/access/Ownable.sol";
-import { ERC20 }   from "openzeppelin-contracts/token/ERC20/ERC20.sol";
+import { IERC20 }  from "openzeppelin-contracts/interfaces/IERC20.sol";
```

Comment:
Should be able to use IERC20Detailed to get access to decimals

Reply:
Good call. Went for `IERC20Metadata`.

Reply:
Though imo it looks cleaner to use `IERC20` where possible and `IERC20Metadata` where necessary. But I can change it to use `IERC20Metadata` everywhere.